public/v4/apps/plausible.yml

```diff
@@ -0,0 +1,131 @@
+captainVersion: 4
+
+services:
+    $$cap_appname-mail:
+        image: bytemark/smtp
+        restart: always
+        environment:
+            RELAY_HOST: $$cap_RELAY_HOST
+            RELAY_PORT: $$cap_RELAY_PORT
+            RELAY_USERNAME: $$cap_RELAY_USERNAME
+            RELAY_PASSWORD: $$cap_RELAY_PASSWORD
+        caproverExtra:
+            notExposeAsWebApp: "true"
+
+    $$cap_appname-postgres:
+        image: postgres:12-alpine
+        volumes:
+            - $$cap_appname-postgres-data:/var/lib/postgresql/data
+        environment:
+            POSTGRES_USER: plausible
+            POSTGRES_PASSWORD: $$cap_POSTGRES_PASSWORD
+        caproverExtra:
+            notExposeAsWebApp: "true"
+
+    $$cap_appname-clickhouse:
+        image: yandex/clickhouse-server:20.8.5.45
+        volumes:
+            - $$cap_appname-clickhouse-data:/var/lib/clickhouse
+        caproverExtra:
+            notExposeAsWebApp: "true"
+
+    $$cap_appname-geoip:
+        image: maxmindinc/geoipupdate:v4.5
+        environment:
+            GEOIPUPDATE_ACCOUNT_ID: $$cap_GEOIPUPDATE_ACCOUNT_ID
+            GEOIPUPDATE_LICENSE_KEY: $$cap_GEOIPUPDATE_LICENSE_KEY
+            GEOIPUPDATE_EDITION_IDS: GeoLite2-Country
+            GEOIPUPDATE_FREQUENCY: 168
+        volumes:
+            - $$cap_appname-geoip-data:/usr/share/GeoIP
+
+    $$cap_appname:
+        depends_on:
+            - $$cap_appname-postgres
+            - $$cap_appname-clickhouse
+            - $$cap_appname-mail
+        caproverExtra:
+            containerHttpPort: "8000"
+            dockerfileLines:
+                - FROM plausible/analytics:v1.1.1
+                - CMD ["sh", "-c", "sleep 10 && /entrypoint.sh db createdb && /entrypoint.sh db migrate && /entrypoint.sh db init-admin && /entrypoint.sh run"]
+        environment:
+            ADMIN_USER_EMAIL: $$cap_ADMIN_USER_EMAIL
+            ADMIN_USER_NAME: $$cap_ADMIN_USER_NAME
+            ADMIN_USER_PWD: $$cap_ADMIN_USER_PWD
+            BASE_URL: http://$$cap_appname.$$cap_root_domain
+            DATABASE_URL: postgres://plausible:$$cap_POSTGRES_PASSWORD@srv-captain--$$cap_appname-postgres:5432/plausible
+            CLICKHOUSE_DATABASE_URL: http://srv-captain--$$cap_appname-clickhouse:8123/plausible
+            SECRET_KEY_BASE: $$cap_gen_random_hex(64)
+            SIGNING_SALT: $$cap_gen_random_hex(24)
+            DISABLE_REGISTRATION: "true"
+            DISABLE_SUBSCRIPTION: "true"
+            MAILER_EMAIL: $$cap_RELAY_USERNAME
+            SMTP_HOST_ADDR: srv-captain--$$cap_appname-mail
+            SMTP_HOST_PORT: "25"
+            GEOLITE2_COUNTRY_DB: "/geoip/GeoLite2-Country.mmdb"
+        volumes:
+            - $$cap_appname-geoip-data:/geoip:ro
```

Comment:
drop `:ro` as it's not respected by CapRover

Reply:
In their page:
https://docs.plausible.io/self-hosting
There are no minimum requirements.
I'll list the different images used in the description.